config/sources/s500.conf

```diff
@@ -13,18 +13,19 @@ GOVERNOR=interactive
 
 case $BOARD in
 	guitar)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-guitar.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-guitar.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 
 	roseapple)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-roseapple.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-roseapple.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 esac
 
 write_uboot_platform()
 {
-	[ -f "$1/s500-bootloader-guitar.bin" ] && dd if=$1/bootloader-guitar.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
-	[ -f "$1/s500-bootloader-roseapple.bin" ] && dd if=$1/bootloader-roseapple.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
+	dd if=$1/$BOOTLOADER_BIN of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
```

Comment:
This won't work as expected because $BOOTLOADER_BIN variable will not be defined in u-boot .deb package postinstall script. DD'ing 1 of 2 files based on their presence should work in any scenario, so please revert changes in this file.